### ISSUE TITLE
Why is office hours on the events page? What does it need?

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/backdropcms.org#1006]
This is a follow-up to https://github.com/backdrop-ops/backdropcms.org/issues/890

A thought: We do have two different google calendars. One is for events, and the other is for meetings. The reason they are separate is because they are managed differently. We did expect anyone to be able to add an event, to do things like promote their own camps or conferences they will be attending. But meetings are "official" and are only added by people who are authorized to create them.

Things like "Office hours" fall directly into this category of "meetings" - things that should not be created by just anyone. We had never planned for meetings to be part of the "events" listing on b.org. Instead, all official (recurring) meetings are documented on [backdropcms.org/support/resources](https://backdropcms.org/support/resources) instead, including a link to the google calendar. 

At some point we started mis-using "events" for office hours, and now it sounds like we are trying to do the same with webinars.

Let's take a step back and see what the office hours need was, and see if we can solve that in another way that also works for webinars. Maybe these things can also auto-feed into our other google calendar (or what I'd prefer, would be feed from the calendar to the website) I'll create a follow-up for that too.

---

@stpaultim correct me if I'm wrong here, but I think the reason we started adding "Office hours" as events was just to increase their visibility. Is that right? We wanted more people to know about them, so we wanted information about them to show up in more places where people might accidentally stumble across it -- is that right? 

If so, then we also need that for Webinars, and we should find a solution that works for both of them.

**Comment by stpaultim** [comment]
@jenlampton 

The simple answer to the question of why office hours is on the events page, is probably the question of visibility. I think, we're trying to recreate a kind of Backdrop CMS calendar, under the assumption (based upon feedback) that very few people really uses the Google Calendar the way it is currently structured. 

We were looking for a way to promote office hours such that someone can come to the site and see that there are office hours on a specific date. I believe this gives people more confidence that they are actually happening. 

We talked today about possibly doing work on the events.backdropcms.org site that would solve this problem. More discussion and work to follow.

**Comment by jenlampton** [comment]
from @irinaz in the parent issue:

> We also need a third sub-task "Make it easier for people to find Backdrop Events". Right now set up is very confusing - [events.backdropcms.org](https://events.backdropcms.org/) only shows BackdropLive - only insiders understand this. If all events are on [backdropcms.org/news/events](https://backdropcms.org/news/events) it would be very helpful to have big block on front page of [events.backdropcms.org](https://events.backdropcms.org/) with link back to events page. Also there are more types of events now, not only office hours and internal meetings.

**Comment by jenlampton** [comment]
> "Make it easier for people to find Backdrop Events".

I don't think that's the complete use-case here. We want people to learn about webinars and office hours when they are *not* looking for information about events. If people don't know they exist, why would they look for them?

> We talked today about possibly doing work on the events.backdropcms.org site that would solve this problem. More discussion and work to follow.

Yes, I saw that suggestion and I also don't think it will help with either use-case. Especially getting people to know about them when they are not looking for them specifically. 

The events site is currently our version of Hopin. It's for registering and attending our own events. It makes sense that it's a separate site, because it has a completely different job than all the others. I'm curious to see the suggested solutions.

**Comment by yorkshire-pudding** [comment]
I agree that Office Hours are much more about support than events or meetings. Attempting to put myself in the shoes of those who have come looking for support, here is a stab at the requirements for Office Hours:

As someone investigating Backdrop as a possibility
I want to understand all the ways I can ask for support
So that I can determine if I will be able to support if I run into difficulty
And if that support will be in a format I can use

As a new site architect trying to build my own Backdrop site
I want to know when the next sessions of Office Hours are
And how to connect
And what it is ok to ask
So that I can find the most convenient time to join and find out how to achieve what I want

As a more experienced site architect or developer but new to Backdrop
I want to get help exploring the best ways to achieve things
So that I understand how to use the API to achieve the results I am after

I think key things to make easy to find:

- dates and times of upcoming sessions in own timezone
- link to join
- what is ok to ask and talk about

I don't think they need to be in events, but they do need to be easy to find from multiple places.

**Comment by stpaultim** [comment]
> We want people to learn about webinars and office hours when they are not looking for information about events. If people don't know they exist, why would they look for them?

I think that people looking for information about events are also interested in community meeting topics and office hours. I also think that sometimes people do know about office hours or community meetings, but don't remember the details or the date/time. If I were coming to BackdropCMS.org looking for information about an upcoming community meeting (also being referred to as webinar) or looking for when office hours are, I would go to the events page.

We might just be using the word event differently here. One might think of events as things like camps, cons, or Backdrop LIVE and think of webinars or office hours as something different. Or, one might think of events as things that happen on the calendar, which would include all of these things. 

Untimately, I think we need a calendar somewhere and I don't think Google Calendar is being used effectively. The reason that Office Hours are listed on the events page, is because it's the best option given the currently available tools. 

At the moment, I treat the events page as a place to post information about things that are happening where people (the public) can come and learn more about Backdrop CMS. I distinguish this from internal meetings, such as the dev meeting or the design/ux meeting. To me, these are something different. That is why I post Office Hours on the events page, but not the weekly dev meeting. Office Hours are specifically designed to promote Backdrop and help people learn about it. 

For me, the point is I want a place for people to go and look for things that they can participate in during the next week or the next month. I want a place, where someone can look if they know that something is scheduled, but they don't remember when it is. 

What I think we need is a community calendar and the current implementation of Google calendar has not proven useful to people. It might be that the events page wasn't intended for this kind of use, but it's the best thing that we have at the moment. 

I think we need something like this: https://www.drupal.org/community/events

**Comment by irinaz** [comment]
1. Office hours are very useful, it is great to see them on https://backdropcms.org/news/events 
2. Do you want to review "Offerings" and change it to "event type" and review this taxonomy to address new landscape for BackDrop events?
3. I prefer layout for Date/Title/Type-of-event that is on  https://www.drupal.org/community/events because it is much easier to scan and date/name of event are closer together and create one block. 
<img width="643" alt="Screenshot 2023-06-30 at 2 51 07 PM" src="https://github.com/backdrop-ops/backdropcms.org/assets/2497005/acc9a2e6-845c-4b9b-94d5-94730d6ec0bd">

What needs to happen to move forward with this issue?  Documented steps?

**Comment by stpaultim** [comment]
I believe that the steps required to move this forward is a very specific proposal that answers the following the questions. 

1) What do we consider events?
2) Where is the best place to create a list of events, on BackdropCMS.org or on the events site?
3) What kind of Google integration do we want/need?

Example: 

I would recommend that we do the following. 

1) Create a chronological display of events on the Events site, we'll need to think about details for how this should look. 
2) Include any public facing gathering targeted at the public as an opportunity to learn about Backdrop CMS as events. This would include camps, BackdropCMS Live, User Group Meetings, Community Meetings (as they are currently defined), and Office Hours. 
3) I would love to see an "Add To My Calendar" button on each event. 

We can continue to treat Meetings differently. Meetings are inward facing and intented largely for the benefit of doing the business of the Backdrop community. The Weekly developer meeting or the UX meeting fit into this. 

I'm certainly open to other variations on this, but am offering this as a more specific starting proposal. Will try to open a new issue with this specific proposal.

**Comment by irinaz** [comment]
@stpaultim @jenlampton @Wylbur @yorkshire-pudding , does that mean that we can add tasks in meta issue #890 or we want to open separate issues?  

Events display already has all information https://backdropcms.org/news/events, just needs formatting more like block in upcoming events in sidebar

**Comment by stpaultim** [comment]
@irinaz - 

I believe that this issue is being blocked by the fact that there are a few different questions that being talked about which effect one another to some degree, but dealing with all three issues at once is creating some confusion. I'm going to open three different discussions to deal with the following three issues. My sense is that this will help us move quicker to improvements.

I am assuming that there is some consensus that we need to figure out a better way of managing event listings. 

1) Should the primary home for our event calendar or event lists be on backdropcms.org or events.backdropcms.org?
https://github.com/backdrop-ops/backdropcms.org/issues/1008
2) What should be included as part of event listings?
https://github.com/backdrop-ops/backdropcms.org/issues/1009
3) What features do we want to include on our event listings page (where ever it lands)?
https://github.com/backdrop-ops/backdropcms.org/issues/1010
